README.md

```diff
@@ -27,10 +27,16 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
 * We recommend you install the [free Anaconda Python distribution](https://www.continuum.io/downloads), which will automatically install Python along with all of the machine learning and scientific computing libraries you need.
 
+* It's a good idea to learn Python's syntax, here are some resources we've used and recommend for learning 'Python for Data Science':
```

Comment:
I think it's a bit obvious that "it's a good idea to learn Python's syntax" ;) perhaps rephrase to: "Here are some resources for getting started with both the Python language in general and Python for data science specifically"